spec/sequel/plugins/enum_values_spec.rb

```diff
@@ -227,6 +227,150 @@
 						-> { expect { subject }.to raise_error(NoMethodError) }
 				end
 			end
+
+			shared_context('with filter_methods environment') do
+				subject(:enum_filter) { item_class.public_send(enum_value) }
+
+				def filter_sql(column, value)
+					"SELECT * FROM \"items\" WHERE (\"#{column}\" = '#{value}')"
+				end
```

Comment:
I think this method can be outside of this shared context: it's not used everywhere where this context is included. And I see no dependency on this context.

Reply:
Or it can be `let` at all. Also, as I see, `value` is always `filter_method_name`.

Reply:
Changed.